crates/ruff_linter/src/rules/pyupgrade/rules/super_call_with_parameters.rs

```diff
@@ -167,9 +167,11 @@ pub(crate) fn super_call_with_parameters(checker: &Checker, call: &ast::ExprCall
 }
 
 /// Returns `true` if a call is an argumented `super` invocation.
-fn is_super_call_with_arguments(call: &ast::ExprCall) -> bool {
+fn is_super_call_with_arguments(call: &ast::ExprCall, checker: &Checker) -> bool {
     if let Expr::Name(ast::ExprName { id, .. }) = call.func.as_ref() {
-        id == "super" && !call.arguments.is_empty()
+        id == "super"
+            && !call.arguments.is_empty()
+            && checker.semantic().match_builtin_expr(&call.func, "super")
     } else {
         false
     }
```

Comment:
Having `is_super_call_with_arguments` should be equivalent and shorter
```rust
checker.semantic().match_builtin_expr(&call.func, "super") && !call.arguments.args.is_empty()
```

Reply:
You're right, thanks!